types/schemas/prototypeApplication/data/Applicant.ts

```diff
@@ -22,124 +27,95 @@ export type BaseApplicant = ContactDetails & {
+       */
+      noticeGiven?: boolean;
+      /**
+       * @description Has a notice of the application been published in a newspaper circulating in the area where the land is situated?
+       */
+      noticePublished?: {
+        status: boolean;
+        date?: Date;
+        newspaperName?: string;
+      };
+      /**
+       * @description Do you know the names and addresses of all owners and agricultural tenants?
+       */
+      ownersKnown?: 'all' | 'some' | 'none';
+      owners?: OwnersNoticeDate[];
+      certificate: 'a' | 'b' | 'c' | 'd';
+      /**
+       * @description Declaration of the accuracy of the ownership certificate, including reasonable steps taken to find all owners and publish notice
+       */
+      declaration: {
+        accurate: true;
+      };
     };
   };
-  /**
-   * @description Contact information for the person(s) responsible for maintenace while the works are carried out
-   */
-  maintenanceContact?: MaintenanceContacts;
-};
-
-export type LandDrainageConsentApplicant = ApplicantBase & {
-  ownership: {
-    interest: OwnersInterest | 'owner.sole' | 'owner.co';
-  };
-  /** @description Contact information for the person(s) responsible for maintenace while the works are carried out */
-  maintenanceContact?: MaintenanceContacts;
-};
-
-export type WTTApplicant = Omit<ApplicantBase, 'siteContact'>;
-
-export type HedgerowRemovalNoticeApplicant = ApplicantBase & {
-  ownership: {
-    interest: OwnersInterest | 'owner.sole' | 'owner.co';
-  };
-};
-
-export type AdvertConsentApplicant = ApplicantBase & {
-  ownership: {
-    interest: OwnersInterest | 'owner.sole' | 'owner.co';
-  };
-};
-
-export type ComplianceConfirmationApplicant = ApplicantBase & {
-  ownership: {
-    interest: OwnersInterest | 'owner.sole' | 'owner.co';
-  };
-};
 
 /**
  * TypeMap of granular application types to their specific Applicant models
  */
 interface ApplicantVariants {
+  advertConsent: ApplicantBase & SiteContact & OwnershipInterest;
+  complianceConfirmation: ApplicantBase & OwnershipInterest;
+  hedgerowRemovalNotice: ApplicantBase & SiteContact & OwnershipInterest;
+  landDrainageConsent: ApplicantBase & MaintenanceContacts & OwnershipInterest;
   'ldc.breachOfCondition': LDCApplicant;
   'ldc.existing': LDCApplicant;
   'ldc.proposed': LDCApplicant;
   'ldc.worksToListedBuilding': LDCApplicant;
-  'pp.full.householder.retro': PPApplicant;
+  listed: PPApplicant;
   'pp.full.householder': PPApplicant;
+  'pp.full.householder.retro': PPApplicant;
   'pp.full.major': PPApplicant;
   'pp.full.minor': PPApplicant;
-  'wtt.consent': WTTApplicant;
-  'wtt.notice': WTTApplicant;
-  advertConsent: AdvertConsentApplicant;
-  complianceConfirmation: ComplianceConfirmationApplicant;
-  hedgerowRemovalNotice: HedgerowRemovalNoticeApplicant;
-  landDrainageConsent: LandDrainageConsentApplicant;
-  listed: PPApplicant;
+  'wtt.consent': ApplicantBase;
+  'wtt.notice': ApplicantBase;
```

Comment:
So this is the big pay off ! 

Our most complicated applicant types can still define their own thing like `LDCApplicant`, `PPApplicant` - but other simple variations can simply "chain" on extra modules like `& SiteContact & MaintenanceContacts` :tada: